### PR TITLE
Topics pages

### PR DESCRIPTION
**Pull request by pablopareja**
## Overview
This PR adds `Topic pages` to the site.
They can be managed in the admin site, a new page has been created.

## Testing instructions
run `yarn install`. New version of `vizz-wysiwyg`has been updated

## Pivotal task
https://www.pivotaltracker.com/story/show/155389817

### CODE REVIEW

File: components/layout/header/header-menu/header-menu-component.js [hunk 1]
```diff
@@ -25,8 +26,9 @@ export default class HeaderMenu extends React.PureComponent {
   headerComponents = {
     data: <HeaderData />,
     about: <HeaderAbout />,
+    topics: <HeaderTopics />,
     myrw: <HeaderUser />,
-    search: <HeaderSearch />
+    search: <HeaderSearch />,
```
Comment (mbarrenechea) [inline components/layout/header/header-menu/header-menu-component.js]: Extra comma

File: components/layout/header/header-topics/component.js [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+import debounce from 'lodash/debounce';
+
+import { Link } from 'routes';
+
+// Components
+import TetherComponent from 'react-tether';
+
+export default class HeaderTopics extends React.PureComponent {
+  toggleDropdown = debounce((bool) => {
+    console.log('hey!');
```
Comment (mbarrenechea) [inline components/layout/header/header-topics/component.js]: hey! oh! lets go!

Comment (pablopareja) [reply components/layout/header/header-topics/component.js]: 😆 It's WIP!

**Comment by pablopareja** [comment]
There's a problem when accessing the Topics page 👉 http://localhost:3000/topics
The browser freezes after a few seconds

**Pull request merged**